stable/chaoskube/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: chaoskube
-version: 0.5.0
+version: 0.6.1
```

Comment:
The `version` should really be the chart version rather than the application version. For the application version you can specify `appVersion`. [Take a look at the wordpress chart for an example](https://github.com/kubernetes/charts/blob/master/stable/wordpress/Chart.yaml).

Reply:
Thanks @mattfarina, I picked `0.6.0` for the chart version (since it was a semantic change) and used `0.6.1` for the `appVersion` (since this is chaoskube's version). Is this correct?